parsec/backend/organization.py

```diff
@@ -94,17 +99,23 @@ async def api_organization_create(self, client_ctx, msg):
 
         bootstrap_token = token_hex(self.bootstrap_token_size)
         expiration_date = msg.get("expiration_date", None)
+        default_users_limit = self._config.organization_config.default_users_limit
         try:
             await self.create(
                 msg["organization_id"],
                 bootstrap_token=bootstrap_token,
                 expiration_date=expiration_date,
+                users_limit=default_users_limit,
             )
 
         except OrganizationAlreadyExistsError:
             return {"status": "already_exists"}
 
-        rep = {"bootstrap_token": bootstrap_token, "status": "ok"}
+        rep = {
+            "users_limit": default_users_limit,
```

Comment:
I don't think returning `users_limit` is needed (the caller will call `organization_status` when it needs this information, and adding this information open the door to add all the other organization configuration stuff...)